### PR TITLE
Hides bot keys from the repository

### PR DESCRIPTION
**Pull request by lennytmp**

### CODE REVIEW

File: Logger.java [hunk 1]
```diff
@@ -120,6 +121,15 @@ static void logException(Exception e) {
     }
   }
 
+  static String getConfigText() throws IOException {
+    String value = null;
```
Comment (vbo) [inline Logger.java]: We have a lot of copies of this code, see e.g. getClient, getIntVar. Maybe make a function for reading the whole file to a string?

File: Logger.java [hunk 2]
```diff
@@ -120,6 +121,15 @@ static void logException(Exception e) {
     }
   }
 
+  static String getConfigText() throws IOException {
```
Comment (vbo) [inline Logger.java]: Why do we throw out? Maybe handle IO errors inside and return null if they happend?

File: Logger.java [hunk 3]
```diff
@@ -120,6 +121,15 @@ static void logException(Exception e) {
...
+    String value = null;
+    FileReader fr = new FileReader(CONFIG_FILE);
+    BufferedReader br = new BufferedReader(fr);
+    value = br.readLine();
```
Comment (vbo) [inline Logger.java]: We have a lot of copies of this code, see e.g. getClient, getIntVar. Maybe make a function for reading the whole file to a string?

**Review by vbo — APPROVED** [review]
Approved with suggestions.

**Comment by lennytmp** [comment]
Thanks for the review! I have done as you instructed, plus refactored the code a little bit, PTAL

File: Logger.java [hunk 4]
```diff
@@ -94,6 +94,17 @@ private static void makeClientsBackup(String[] filenames) {
     }
   }
 
+  private static String readOneLineFile(String filename) {
+    String value = null;
+    try (FileReader fr = new FileReader(filename);
```
Comment (vbo) [inline Logger.java]: fr is unused. Maybe var br = new BufferedReader(new FileReader(filename))?

**Review by vbo — APPROVED** [review]
Approved with suggestions.

**Pull request merged**